PED/GivePedHelmet.md

```diff
@@ -8,19 +8,11 @@ ns: PED
 void GIVE_PED_HELMET(Ped ped, BOOL cannotRemove, int helmetFlag, int textureIndex);
 ```
 
-```
-PoliceMotorcycleHelmet   1024
-RegularMotorcycleHelmet   4096
-FiremanHelmet 16384
-PilotHeadset  32768
-PilotHelmet   65536
```

Comment:
Thought I would say, the texture helmet indexes are kind of useful to have.

Reply:
Should be good.